ingestion/src/metadata/utils/sql_lineage.py

```diff
@@ -13,33 +13,87 @@
 """
 import traceback
 from logging.config import DictConfigurator
+from typing import List, Optional
 
 from metadata.generated.schema.api.lineage.addLineage import AddLineageRequest
 from metadata.generated.schema.entity.data.table import Table
-from metadata.generated.schema.type.entityLineage import EntitiesEdge
+from metadata.generated.schema.type.entityLineage import (
+    ColumnLineage,
+    EntitiesEdge,
+    LineageDetails,
+)
 from metadata.generated.schema.type.entityReference import EntityReference
 from metadata.ingestion.ometa.ometa_api import OpenMetadata
 from metadata.utils import fqn
 from metadata.utils.helpers import _get_formmated_table_name
 from metadata.utils.logger import utils_logger
 
-
-# Prevent sqllineage from modifying the logger config
-def configure(self):
-    pass
-
-
-DictConfigurator.configure = configure
-
-
 logger = utils_logger()
+column_lineage_map = {}
 
 
 def _separate_fqn(database, fqn):
     database_schema, table = fqn.split(".")[-2:]
     if not database_schema:
         database_schema = None
-    return {"database": database, "database_schema": database_schema, "name": table}
+    return {"database": database, "database_schema": database_schema, "table": table}
+
+
+def get_column_fqn(table_entity: Table, column: str) -> Optional[str]:
+    """
+    Get fqn of column if exist in table entity
+    """
+    for tbl_column in table_entity.columns:
+        if column.lower() == tbl_column.name.__root__.lower():
+            return tbl_column.fullyQualifiedName.__root__
+
+
+def _get_table_entities(
```

Comment:
we have something super similar in another place. Let's try to have a single util using fqn.build. This already searches in ES